plugin/src/test/java/com/yelp/plugin/PluginTests.kt

```diff
@@ -0,0 +1,28 @@
+package com.yelp.plugin
+
+import org.gradle.testkit.runner.BuildResult
+import org.gradle.testkit.runner.GradleRunner
+import org.gradle.testkit.runner.TaskOutcome
+import org.junit.Assert
+import org.junit.Test
+import java.io.File
+
+
+class PluginTests {
+  @Test
+  fun `basic plugin test`() {
```

Comment:
```suggestion
  fun basicPluginTest() {
```

Let's stay consistent with the other tests 👍